great_expectations/jupyter_ux/expectation_explorer.py

```diff
@@ -417,58 +1379,201 @@ def create_expectation_widget(
+                <hr>
+                <ul>
+                    <li><strong>Data Asset Name</strong>: {data_asset_name}</li>
+                    <li><strong>Great Expectations Version</strong>: {ge_version}</li>
+                    <li><strong>Column Count</strong>: {column_count}</li>
+                    <li><strong>Expectation Count</strong>: {expectation_count}</li>
+                </ul>
+                <hr>
+                <h2>Columns:</h2>
+            </div>\
+        """.format(
+            data_asset_name=data_asset_name,
+            ge_version=ge_version,
+            column_count=column_count,
+            expectation_count=expectation_count
+        )
+        summary_widget = widgets.HTML(
+            value=summary_widget_content,
+            layout=widgets.Layout(
+                margin='10px',
+            )
+        )
+
+        for column_name in column_names:
+            expectation_editor_widgets = []
+            column_expectations = data_asset_expectations[column_name]
+            expectation_types = list(column_expectations.keys())
+            expectation_types.sort()
+            for expectation_type in expectation_types:
+                editor_widget = column_expectations[expectation_type]['editor_widget']
+                editor_widget.selected_index = None
+                expectation_editor_widgets.append(editor_widget)
+            column_accordion_content = widgets.VBox(
+                children=expectation_editor_widgets)
+            column_accordion = widgets.Accordion(
+                children=[column_accordion_content])
+            column_accordion.set_title(0, column_name)
+            column_accordion.selected_index = None
+            column_accordions.append(column_accordion)
+
+        return [summary_widget] + column_accordions
+
+    def edit_expectation_suite(self, data_asset):
+        data_asset_name = data_asset.get_data_asset_name()
+        expectation_suite = data_asset.get_expectation_suite(
+            discard_failed_expectations=False)
+        expectations = expectation_suite.get('expectations')
+
+        ################### editor widgets
+        for expectation in expectations:
+            expectation_type = expectation.get('expectation_type')
+            expectation_kwargs = expectation.get('kwargs')
+            editor_widget = getattr(data_asset, expectation_type)(
+                include_config=True, **expectation_kwargs)
+
+        expectation_suite_editor_widgets = self.generate_expectation_suite_editor_widgets(data_asset, expectation_suite)
+        ###################
+        expectation_suite_editor = widgets.VBox(
+            children=expectation_suite_editor_widgets,
+            layout=widgets.Layout(
+                border='2px solid black',
+            )
+        )
+
+        self.state['data_assets'][data_asset_name]['expectation_suite_editor'] = expectation_suite_editor
+
+        return expectation_suite_editor
```

Comment:
Did you just create a whole notebooks-driven survey editor? I'm so jazzed to start to play with this. I bet @Aylr will have a bunch of fun ideas...